rdmo/core/tests/test_project_status.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management import call_command
+import pytest
+
+def test_makemigrations_has_no_changes(db, capsys):
+    call_command("makemigrations", check=True, dry_run=True)
+    captured = capsys.readouterr()
+    assert "No changes detected" in captured.out
+
+
+# TODO
+@pytest.mark.skip(reason="test failing on CI")
+def test_compilemessages_is_already_up_to_date(capsys):
```

Comment:
This test is not working as expected. Should I remove it?
We discussed the "compilemessages" issue today.

Reply:
`nl` is not translated yet, just an empty `.po` file,
 it can be left out and then it should pass or?

Reply:
I see, let me try that. I was confused, because locally the test passed. Will report back to you.

Reply:
Locally I get this output for every language, including "nl":

 'File “[...]/rdmo/locale/{language}/LC_MESSAGES/django.po” is already compiled and up to date.'

That was the basis of my initial test. So I was confused, when this failed in CI.

Reply:
yes, only at the initial run of  `compilemessages` it gives the `processing file django.po in ...` for `nl`

Reply:
I still don't get it. :)